src/client/util/http.cc

```diff
@@ -111,6 +112,8 @@ bool HTTP::DownloadViaClearnet() {
   // Create and set options
   Options options;
   options.timeout(static_cast<std::uint8_t>(Timeout::Request));
+  LOG(debug) << "HTTP: Download Clearnet with timeout : "
+             << std::to_string(static_cast<std::uint8_t>(Timeout::Request));
```

Comment:
Some alternatives:

- `#include "core/util/byte_stream.h"` `kovri::core::GetType(Timeout::Request)`
- Refactor `Timeout` as it currently doesn't need to be scoped *if* refactored as a member of class `HTTP`, then simply use `Timeout::Request`